tests/Equinox.EventStore.Integration/StoreIntegration.fs

```diff
@@ -503,27 +557,118 @@ type Tests(testOutputHelper) =
         test <@ singleBatchBackwards @ batchBackwardsAndAppend @ suboptimalExtraSlice @ singleBatchForward = capture.ExternalCalls @>
     }
 #endif
+
+
+#if STORE_MESSAGEDB
+type AdjacentSnapshotTests(testOutputHelper) =
+    let sdk =
+        Sdk.CreateTracerProviderBuilder()
+           .SetResourceBuilder(ResourceBuilder.CreateDefault().AddService(serviceName = "mdbi"))
+           .AddSource("Equinox")
+           .AddSource("Equinox.MessageDb")
+           .AddSource("StoreIntegration")
+           .AddSource("Npqsl")
+           .AddOtlpExporter(fun opts -> opts.Endpoint <- Uri("http://localhost:4317"))
+           .Build()
+
+    let output = TestContext(testOutputHelper)
+
+    let sliceForward = [EsAct.SliceForward]
+    let singleBatchForward = [EsAct.SliceForward; EsAct.BatchForward]
+    let readSnapshotted = [EsAct.ReadLast; EsAct.SliceForward; EsAct.BatchForward]
+
     [<AutoData(SkipIfRequestedViaEnvironmentVariable="EQUINOX_INTEGRATION_SKIP_EVENTSTORE")>]
```

Comment:
maybe this can now be to be `EQUINOX_INTEGRATION_SKIP_MESSAGEDB`, with associated def of that in the CI scripts

Reply:
It probably can be. Though since the purpose is to exclude from CI, I think we might as well change the variable to "CI" as that's set automatically in azure and github actions.

Reply:
Suggestion: Let's keep as is, and my next PR will rig CI for message-db and SSS PG?

Reply:
Just conscious of not doing too much in the span of a single PR

Reply:
no arguments of any kind :D Also no pressure...